go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -650,7 +655,7 @@ func (be *BuiltinBackupEngine) restoreFile(ctx context.Context, params RestorePa
 		}
 	}()
 
-	bp := newBackupReader(name, source)
+	bp := newBackupReader(name, 0, source)
```

Comment:
what does it mean to pass maxSize as 0?

Reply:
> what does it mean to pass maxSize as 0?

It was getting set to 0 implicitly before now we have to explicitly do it since at other places we do pass size to reader now.